src/models/neural_networks/rnn.py

```diff
@@ -136,12 +142,14 @@ def forward(self, x, pred_month=None, current=None):
 
         x = hidden_state.squeeze(0)
 
+        if self.include_pred_month:
```

Comment:
Yea - the assumption here is that if `RNN.include_latlons` is True, then it must have been passed because that means `RecurrentNetwork.include_latlons` must also be True